packages/adapter-nuxt/src/hooks/documentation-read.ts

```diff
@@ -39,12 +39,13 @@ export const documentationRead: DocumentationReadHook<PluginOptions> = async (
 					})
 				);
 
-				useHead({
+				useSeoMeta({
 					title: page.value?.data.meta_title,
-					meta: [{
-						name: "description",
-						content: page.value?.data.meta_description,
-					}],
+					ogTitle: page.value?.data.meta_title,
```

Comment:
❓ Why having the `og*` meta tags now as compared to before?

If it's something we want to have now? So we should add it to our other adapters, correct?

Reply:
Replied here: https://github.com/prismicio/slice-machine/pull/1526#discussion_r1901727387